prompting/utils/config.py

```diff
@@ -83,7 +83,7 @@ def add_args(cls, parser):
         "--neuron.llm_max_allowed_memory_in_gb",
         type=int,
         help="The max gpu memory utilization set for initializing the model. This parameter currently reflects on the property `gpu_memory_utilization` of vllm",
-        default=60,
+        default=80,
```

Comment:
Why are we setting default at 80GB? 
I've had bad experiences previously by setting `gpu_memory_utilization` very close to hardware limit, I got OOM often, and we don't actually need all that to run llama3 awq 4 bits

Reply:
> Why are we setting default at 80GB? I've had bad experiences previously by setting `gpu_memory_utilization` very close to hardware limit, I got OOM often, and we don't actually need all that to run llama3 70B awq 4 bits

For me it's vice-versa, with default `--neuron.llm_max_allowed_memory_in_gb 60` on 2xA40 (96GB total), I got:
```
ValueError: No available memory for the cache blocks. Try increasing gpu_memory_utilization when initializing the engine.
```

Just checked, actually minimum value of `65` works well.